app/mailers/application_mailer.rb

```diff
@@ -0,0 +1,4 @@
+class ApplicationMailer < ActionMailer::Base
```

Comment:
Whats this file about?

Reply:
This is our base class for all mailers. This will be inherited by all mailers. This is where you can define common methods and defaults, e.g. the sender email. Mailer allows you to send emails from your application using mailer classes and views.